### ISSUE TITLE
re Service Discovery

### ISSUE THREAD

**Issue by cppforlife** [starkandwayne/ultimate-guide-to-bosh#17]
> https://ultimateguidetobosh.com/service-discovery/

I wouldn't separate BOSH DNS from BOSH Links. BOSH Links provides service discovery mechanism, and links take advantage of DNS when it's enabled. BOSH DNS on its own is more about internal addressability rather than service discovery.

**Comment by drnic** [comment]
Yeah I’m not yet sure I like this section as it stands/where it’s going.
